lib/lita/handlers/jenkins.rb

```diff
@@ -1,6 +1,13 @@
 require 'json'
 require 'base64'
 
+# Determine if a given String is contains only a number.
+class String
```

Comment:
Please [don't monkey patch](http://devblog.avdi.org/2008/02/23/why-monkeypatching-is-destroying-ruby/) core classes. Here's [one option](http://www.justinweiss.com/blog/2015/01/20/3-ways-to-monkey-patch-without-making-a-mess/), or another might be [refinements](http://ruby-doc.org/core-2.1.1/doc/syntax/refinements_rdoc.html) (I haven't really looked into refinements yet myself tho).